### PR TITLE
T1-3 Merge branch 'T1-3' of github.com:mpaz-redstage/bot-test into T1-3

### PR DESCRIPTION
**Pull request by mpaz-redstage**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # bot-test
+
+this is a new change
```
Comment (mpaz-redstage) [inline README.md:3]:
```suggestion
this is a new change !
```

Comment (mpaz-redstage) [reply README.md:3]: e

Comment (mpaz-redstage) [reply README.md:3]: asdf

Comment (mpaz-redstage) [reply README.md:3]: teste

Comment (mpaz-redstage) [reply README.md:3]: 2

Comment (mpaz-redstage) [reply README.md:3]: d

Comment (mpaz-redstage) [reply README.md:3]: d

Comment (mpaz-redstage) [reply README.md:3]: d

Comment (mpaz-redstage) [reply README.md:3]: d

Comment (mpaz-redstage) [reply README.md:3]: E

Comment (mpaz-redstage) [reply README.md:3]: teste

**Review by mpaz-redstage — COMMENTED** [review]
teste

File: README.md (hunk 1 above)
Comment (mauriciopazpp) [reply README.md:3]: teste 2

Comment (mpaz-redstage) [reply README.md:3]: teste!

Comment (mpaz-redstage) [reply README.md:3]: teste

Comment (mpaz-redstage) [reply README.md:3]: teste

Comment (mpaz-redstage) [reply README.md:3]: teste

Comment (mpaz-redstage) [reply README.md:3]: teste

Comment (mpaz-redstage) [reply README.md:3]: teste

Comment (mpaz-redstage) [reply README.md:3]: teste!

Comment (mpaz-redstage) [reply README.md:3]: teste

Comment (mpaz-redstage) [reply README.md:3]: teste

Comment (mpaz-redstage) [reply README.md:3]: teste

**Review by mauriciopazpp — APPROVED** [review]

**Review by mauriciopazpp — APPROVED** [review]

**Review by mauriciopazpp — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mpaz-redstage) [reply README.md:3]: teste

**Review by mauriciopazpp — APPROVED** [review]

**Review by mauriciopazpp — APPROVED** [review]

**Review by mauriciopazpp — APPROVED** [review]

**Review by mauriciopazpp — CHANGES_REQUESTED** [review]
teste

File: README.md (hunk 1 above)
Comment (mpaz-redstage) [reply README.md:3]: this is a new comment

**Pull request closed without merging**